iis3dwb_reg.c

```diff
@@ -424,13 +424,14 @@ int32_t iis3dwb_xl_axis_selection_get(stmdev_ctx_t *ctx,
 {
   iis3dwb_ctrl4_c_t ctrl4_c={0};
   iis3dwb_ctrl6_c_t ctrl6_c={0};

```

Comment:
Is this initialization strictly required to avoid static analysis issue? Because they are overwritten by reads below, and now if read operation fails we have the proper return.

Reply:
Fixed.
And squashed all the commits